tools/go.mod

```diff
@@ -45,7 +46,7 @@ require (
 	github.com/Azure/go-autorest/logger v0.2.1 // indirect
 	github.com/Azure/go-autorest/tracing v0.6.0 // indirect
 	github.com/AzureAD/microsoft-authentication-library-for-go v1.0.0 // indirect
-	github.com/BurntSushi/toml v1.3.2 // indirect
+  github.com/BurntSushi/toml v1.3.2 // indirect
```

Comment:
these `tools/go.{mod,sum}` changes look unintentional. let's remove them from the PR

Reply:
They are indeed, I was resolving a merge conflict in haste.